server/src/main/java/org/opensearch/env/NodeEnvironment.java

```diff
@@ -1306,4 +1421,30 @@ private static void tryWriteTempFile(Path path) throws IOException {
             }
         }
     }
+
+    /**
+     * Returns the {@link FileCache} instance for remote search node
+     */
+    public FileCache remoteStoreFileCache() {
+        return this.remoteStoreFileCache;
+    }
+
+    /**
+     * Returns the current {@link FileCacheStats} for remote search node
+     */
+    public FileCacheStats remoteStoreFileCacheStats() {
```

Comment:
Down the line, I think we should have a FCMonitorService to keep track of the cache stats. By design, stat calls do not logically belong within the NodeEnvironment instance.